src/tvheadend/entity/Recording.h

```diff
@@ -60,7 +60,8 @@ namespace tvheadend
         m_stopExtra(0),
         m_state(PVR_TIMER_STATE_ERROR),
         m_retention(99), // Kodi default - "99 days"
-        m_priority(50) // Kodi default - "normal"
+        m_removal(-1),   // max value to stay compatible with older backends
+        m_priority(50)   // Kodi default - "normal"
```

Comment:
Actually I don't think that these values are needed, they are being overwritten by the actual (htsp) values? @ksooo